modules/microshift-embed-images-offline-use.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-embed-images-offline-use_{context}"]
+= Embedding workload container images for offline use
+
+To embed container images in devices at the edge that do not have any network connection, you must create a new container, mount the ISO, and then copy the contents into the file system.
+
+.Prerequisites
+
+* You have root access to the host.
+* Application RPMs have been added to a blueprint.
+
+.Procedure
+
+. Render the manifests, extract all of the container image references, and translate the application image to blueprint container sources by running the following command:
++
+[source,terminal]
+----
+$ oc kustomize ~/manifests | grep "image:" | grep -oE '[^ ]+$' | while read line; do echo -e "[[containers]]\nsource = \"${line}\"\n"; done >><my-blueprint>.toml
+----
+
+. Push the updated blueprint to Image Builder by running the following command:
++
+[source, terminal]
+----
+$ sudo composer-cli blueprints push <my-blueprint>.toml
```

Comment:
```suggestion
$ sudo composer-cli blueprints push <my_blueprint>.toml
```

Reply:
One day I will remember this. Got the one above, too.